<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinphoneWindow</class>
 <widget class="QMainWindow" name="LinphoneWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LinphoneWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QWidget" name="leftSideWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="searchBarLayout">
           <item>
            <widget class="QLineEdit" name="searchBar">
             <property name="placeholderText">
              <string>Add a conversation...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addConversationBtn">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QWidget" name="accountWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="accountLayout">
              <item>
               <widget class="QGraphicsView" name="accountStatus">
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="foregroundBrush">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>82</red>
                   <green>204</green>
                   <blue>83</blue>
                  </color>
                 </brush>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="accountCombo">
                <property name="currentText">
                 <string/>
                </property>
                <property name="duplicatesEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="accountOptions">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QListView" name="conversationsList"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="chatList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
